lighthouse-core/test/audits/metrics/first-contentful-paint-test.js

```diff
@@ -35,16 +38,32 @@ describe('Performance: first-contentful-paint audit', () => {
   it('evaluates valid input correctly', async () => {
     const artifacts = {
       traces: {
-        [Audit.DEFAULT_PASS]: pwaTrace,
+        [FcpAudit.DEFAULT_PASS]: pwaTrace,
       },
       devtoolsLogs: {
-        [Audit.DEFAULT_PASS]: pwaDevtoolsLog,
+        [FcpAudit.DEFAULT_PASS]: pwaDevtoolsLog,
       },
     };
 
     const context = getFakeContext({formFactor: 'mobile', throttlingMethod: 'provided'});
-    const result = await Audit.audit(artifacts, context);
+    const result = await FcpAudit.audit(artifacts, context);
     assert.equal(result.score, 1);
     assert.equal(result.numericValue, 498.87);
   });
+
+  it('evaluates a modern trace correctly', async () => {
+    const artifacts = {
+      traces: {
+        [FcpAudit.DEFAULT_PASS]: frameTrace,
+      },
+      devtoolsLogs: {
+        [FcpAudit.DEFAULT_PASS]: frameDevtoolsLog,
+      },
+    };
+
+    const context = getFakeContext({formFactor: 'mobile', throttlingMethod: 'provided'});
+    const result = await FcpAudit.audit(artifacts, context);
+    assert.equal(result.score, 0.06);
+    assert.equal(result.numericValue, 5668.275);
+  });
```

Comment:
we didn't have any unit test that needed updating from the score change (aside from sample_v2 snapshot ones), which is always suspicious, so added one here